src/client/googlePubSub.ts

```diff
@@ -1,15 +1,15 @@
 import util from 'util';
-import chalk from 'chalk';
 import {
   PubSub as GooglePubSub,
   Message as GoogleCloudMessage,
   Subscription as GoogleCloudSubscription,
   Topic as GoogleCloudTopic,
 } from '@google-cloud/pubsub';
 import { Resource } from '@google-cloud/resource';
-
 import { CredentialBody } from 'google-auth-library';
+import chalk from 'chalk';
```

Comment:
Codacy has a fix for the issue: [Replace `chalk·from·'chalk` with `util·from·'util`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
import util from 'util';
```